src/main/java/com/northconcepts/datapipeline/foundations/examples/datamapping/DeclarativelyExcludeFieldsAndMapDataAutomatically.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2006-2023 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential.  Use is subject to license terms.
+ * 
+ * https://northconcepts.com/data-pipeline/licensing/
+ */
+package com.northconcepts.datapipeline.foundations.examples.datamapping;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.foundations.datamapping.DataMapping;
+import com.northconcepts.datapipeline.foundations.datamapping.DataMappingReader;
+import com.northconcepts.datapipeline.job.Job;
+
+import java.io.File;
+import java.io.FileInputStream;
+
+public class DeclarativelyExcludeFieldsAndMapDataAutomatically {
```

Comment:
Let's switch exclude and map around into the order that it actually happens: `AutomaticallyMapDataAndExcludeFieldsDeclaratively`

Reply:
Done